pkgs/development/haskell-modules/configuration-arm.nix

```diff
@@ -38,6 +38,11 @@ self: super: {
   happy = dontCheck super.happy;
   happy_1_19_12 = doDistribute (dontCheck super.happy_1_19_12);
 
+  # add arm specific library
+  wiringPi = overrideCabal ({librarySystemDepends ? [], ...}: {
+    librarySystemDepends = librarySystemDepends ++ [pkgs.wiringpi];
```

Comment:
Why is this only needed on aarch?  I'm kinda surprised, I would have thought it would have been needed on all platforms, and it would be picked up automatically.

Also, we generally ask for Haskell-related PRs to go to the `haskell-updates` branch.

Reply:
The wiringPi.cabal file only adds wiringpi as a dependency on aarch and I will target the haskell-updates branch instead.

see https://github.com/ppelleti/hs-wiringPi/blob/master/wiringPi.cabal#L95-L96

Reply:
It appears to use a dummy IO implementation on x86 platforms

Reply:
@MaxHearnden Ah, I see.  Thanks for the explanation, that makes sense.

@ofborg build haskellPackages.wiringPi